.gitignore

```diff
@@ -26,6 +26,8 @@ dist/
 # doc generated
 docs/_build
 wiki/
+docs/core/beam.gif
+docs/core/beam_wireframe.gif
```

Comment:
These might have been left around from an old build.  I've reworked the docbuild using `jupyter-execute` rather than the `test-code` directive.  It's really effective, and even means we can have interactive docs (see https://dev.pyvista.org/jupyter/index.html), but it does require a bit of extra work to move some of the generated animations.

I don't think removing or keeping these should matter.